docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3795,34 +3803,56 @@ the `GetForwardingPipelineConfig RPC`. The request is defined as:
 
 ~ Begin Proto
 message GetForwardingPipelineConfigRequest {
+  enum ResponseType {
+    ALL = 0;
+    COOKIE_ONLY = 1;
+    P4INFO_ONLY = 2;
```

Comment:
Should the controller have the option to get the p4_device_config only, without the p4info?

Reply:
There isn't much harm in supporting this mode as well. However I don't think there is really a requirement for this case. @ccascone's concern - that lead to this proposed change - was the potential size of the `p4_device_config` on some targets.

Reply:
I added a `DEVICE_CONFIG_AND_COOKIE`. I can envision cases in which the device config has a size smaller than the p4info, e.g. for a target that can support only a limited set of possible device config / P4 programs, the config field might be used to encode the ID of one of these configs already available in the device.

Reply:
Still unconvinced that this is a useful mode to have (even for small device configs). However it is not any kind of burden to implement so I don't have any strong objection.